src/sentry/api/helpers/deprecation.py

```diff
@@ -27,18 +32,56 @@ def _track_deprecated_metrics(request: Request, deprecation_date: datetime):
     request.deprecation_date = deprecation_date
 
 
-def _should_be_blocked(deprecation_date: datetime, now: datetime):
+def _serialize_key(key: str) -> Tuple[str, str]:
+    """Converts the key into an option manager key used for the schedule crontab and blackout duration"""
+    return f"{key}-cron", f"{key}-duration"
+
+
+def _should_be_blocked(deprecation_date: datetime, now: datetime, key: str):
+    """
+    Determines if a request should be blocked given the current date time and a schedule
+
+    For example if the schedule blocks requests at noon for 1 minute, then a request coming in at
+    11:59:59 would be allowed while one at 12:00:01 would not
+    """
     # Will need to check redis if the hour fits into the brownout period
     if now >= deprecation_date:
+        key = "api.deprecation.brownout" if not key else key
```

Comment:
I think this line can just be removed by just setting the default at the parameter.

Reply:
`key` is being passed in from the decorator
```
_should_be_blocked(deprecation-date, now, key)
```
So the parameter default would never be used as technically something is being passed in.